tox.ini

```diff
@@ -12,6 +12,15 @@ commands =
     sh -c 'rm -f $HOME/.matplotlib/fontList*'
     {envpython} {toxinidir}/tests.py --processes=-1 --process-timeout=300
 deps =
-    nose
+    pytest
     mock
     numpy
+
+[pytest]
+norecursedirs = .git build
+testpaths = tests
+pep8ignore = E111 E114 E115 E116 E121 E122 E123 E124 E125 E126 E127 E128 E129 E131 E265 E266 W503 E241 E401 E231 E203 E226
+
+[pep8]
+ignore=E111,E114,E115,E116,E121,E122,E123,E124,E125,E126,E127,E128,E129,E131,E265,E266,W503
```

Comment:
It seems you have two different methods to list ignore PEP8 violations here...  What's the difference?